frontend/www/js/omegaup/components/arena/ClarificationList.vue

```diff
@@ -134,8 +150,12 @@ export default class ArenaClarificationList extends Vue {
   @Prop() username!: null | string;
   @Prop({ default: false }) showNewClarificationPopup!: boolean;
   @Prop({ default: false }) allowFilterByAssignment!: boolean;
+  @Prop() length!: number;
```

Comment:
nit:

```suggestion
  @Prop() pageSize!: number;
```

me costó un poco de trabajo darme cuenta que este era el tamaño de la página ^^;;

y ahora que lo noto, muchas cosas en `Course.php` usan `length` en vez de `pageSize` :/ uno de estos días tenemos que normalizar eso.

Reply:
Sí, también noté que no hay una uniformidad en eso, a veces en los controladores y DAOs usamos `rowcount` y a veces `length` y otras `pageSize`...